setup.py

```diff
@@ -103,10 +103,11 @@ def txt_read(*paths):
 setup_args['install_requires'] =\
     [
         "numpy",
-        "matplotlib",
+        "matplotlib>=1.5",
+        "cartopy",
```

Comment:
So ``cartopy[plotting]`` would add ``default.txt`` + ``plotting.txt`` requirements.. is this correct?